platform/ui/src/components/ThumbnailTracked/ThumbnailTracked.tsx

```diff
@@ -87,12 +87,12 @@ function ThumbnailTracked ({
                 </div>
                 <div className="flex flex-1 flex-col">
                   <span>
-                    <span className="text-white">{isTracked ? t('Series is tracked') : t('Series is untracked')}</span>
+                    <span className="text-white">
```

Comment:
This class to me feels like it belongs in an extension, not in platform/ui because it is dealing with mode specific components/information.  Not sure I feel like asking you to move it, but a comment about it might be appropriate.